Sources/Views/MessageInputBar.swift

```diff
@@ -421,7 +421,7 @@ open class MessageInputBar: UIView {
     open func calculateIntrinsicContentSize() -> CGSize {
         
         let maxTextViewSize = CGSize(width: inputTextView.bounds.width, height: .greatestFiniteMagnitude)
-        var heightToFit = inputTextView.sizeThatFits(maxTextViewSize).height.rounded()
```

Comment:
I think it should be `rounded(.up)` right?

Reply:
@SD10 ,yeah, default is `.toNearestOrAwayFromZero`, the `height` is not negative I think , so it's equal to `.up`?